xnmt/embedder.py

```diff
@@ -143,3 +144,72 @@ def embed_sent(self, sent):
           embeddings.append(self.embed(xnmt.batcher.mark_as_batch([single_sent[word_i] for single_sent in sent])))
       return ExpressionSequence(expr_list=embeddings, mask=sent.mask)
 
+
+class PretrainedSimpleWordEmbedder(SimpleWordEmbedder):
```

Comment:
I think it'd be better to just add an option to let it read in embeddings from a file if the argument is not None, rather than creating another class. What do you think?

Reply:
I agree; the only argument I could see for keeping a separate class is if we want to add some different logic to prevent pretrained weights from being updated / being changed too much.

Reply:
Ahh, that's a good point. Actually now that you mention it, we might want to keep this as-is so we can add this functionality later.